boards/arm/nucleo_wl55jc/nucleo_wl55jc.dts

```diff
@@ -169,6 +169,13 @@
 		rx-enable-gpios = <&gpioc 5 GPIO_ACTIVE_LOW>;
 		dio3-tcxo-voltage = <SX126X_DIO3_TCXO_1V7>;
 		tcxo-power-startup-delay-ms = <5>;
+		/* Hard-code the high-power output, which is selected
```

Comment:
I would reword this to make it clear that we are not hardcoding the HP output, but it is hardcoded because we are (ab)using the `tx/rx-enable-gpio` to control `FE_CTRL1/2`,
```
High-power output is selected as a consequence of using tx/rx-enable-gpio
to control FE_CTRL1 and FE_CTRL2. Low-power output would require both
FE_CTRL1 and FE_CTRL2 to be high, which is not currently supported by the
driver.
```
I would also add a `FE_CTRL1` and `FE_CTRL2` comment on the two GPIO's above to make it as clear as possible.

Reply:
That wording is much clearer, will update it thanks.